gym/wrappers/rescale_action.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+import gym
+from gym import spaces
+
+
+class RescaleAction(gym.ActionWrapper):
+    r"""Rescale the continuous action space of the environment from a range of [a, b]. """
```

Comment:
The docstring sounds a little confusing to me. How about somewhat more wordy version:
```
Rescales the continuous action space of the environment to a range [a,b], such that
RescaleAction(env, a,b).action_space == Box(a,b)
```
?

Reply:
@pzhokhov Thanks for the comment, updated accordingly.